examples/resources/genesyscloud_credential/resource.tf

```diff
@@ -0,0 +1,9 @@
+resource "genesyscloud_credential" "credential1" {
```

Comment:
Can we make this `genesyscloud_integration_credential` instead to match the API path?

Reply:
Name Changed